src/Core/MailTemplates/Handlebars/InvoiceUpcoming.html.hbs

```diff
@@ -17,7 +17,14 @@
     {{/if}}
     <tr style="margin: 0; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
         <td class="content-block" style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; margin: 0; -webkit-font-smoothing: antialiased; padding: 0 0 10px; -webkit-text-size-adjust: none;" valign="top">
-            To avoid any interruption in service, please ensure that your payment method on file is up to date and can be charged for the above amount. You can manage your subscription, payment method, and invoices by logging into the web vault at {{{link WebVaultUrl}}}.
+            To avoid any interruption in service, please ensure that your payment method on file is up to date and can be charged for the above amount. You can manage your subscription, payment method, and invoices by logging into the web vault.
+            <a href="https://bitwarden.com/help/update-billing-info/" class="text-decoration-none">
```

Comment:
Please put this URL on the view model (`InvoiceUpcomingViewModel`) and use the handlebars syntax to insert it into the template. (e.g. `{{{ link UpdateBillingInfoUrl }}}`)

It's tidier, and we only have to maintain it in 1 place (the view model) instead of updating both templates if the URL changes.